apps/profiler-ui/src/docs.ts

```
@@ -29,8 +29,8 @@
 // All documents are static (profiler sessions per connection) — no VFS needed.
 // =============================================================================
 
-import { Documents } from 'shell-ui';
-import type { WorkspaceBinding } from 'shell-ui';
+import { Documents } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)